tutorials/prep.md

```diff
@@ -1,10 +1,88 @@
+- *`FAVOURED_PREP_TYPE`* - The `PrEPType` with the highest preference value an individual is willing to take that is also currently available. If there is no PrEP available that they are willing to take, this value is set to None.
+- *`PREP_ELIGIBLE`* - A boolean flag signifying whether an individual is eligible for PrEP this time step.
+- *`PREP_ORAL_TESTED`* - A boolean flag signifying whether an individual has tested for HIV specifically for the purpose of starting oral PrEP. `Note`: Currently dummied.
+- *`PREP_CAB_TESTED`* - A boolean flag signifying whether an individual has tested for HIV specifically for the purpose of starting cabotegravir PrEP. `Note`: Currently dummied.
+- *`PREP_LEN_TESTED`* - A boolean flag signifying whether an individual has tested for HIV specifically for the purpose of starting lenacapavir PrEP. `Note`: Currently dummied.
+- *`PREP_VR_TESTED`* - A boolean flag signifying whether an individual has tested for HIV specifically for the purpose of starting vaginal ring PrEP. `Note`: Currently dummied.
+- *`PREP_TYPE`* - The most recent `PrEPType` an individual has used, otherwise None if they have never used PrEP. This column is kept intact upon stopping PrEP usage.
+- *`EVER_PREP`* - A boolean flag signifying whether an individual has ever been on PrEP.
+- *`FIRST_ORAL_START_DATE`* - The start date of an individual's first ever usage of oral PrEP, otherwise None if they have never used oral PrEP.
+- *`FIRST_CAB_START_DATE`* - The start date of an individual's first ever usage of cabotegravir PrEP, otherwise None if they have never used cab PrEP.
+- *`FIRST_LEN_START_DATE`* - The start date of an individual's first ever usage of lenacapavir PrEP, otherwise None if they have never used len PrEP.
+- *`FIRST_VR_START_DATE`* - The start date of an individual's first ever usage of vaginal ring PrEP, otherwise None if they have never used vr PrEP.
+- *`LAST_PREP_START_DATE`* - The start date of an individual's most recent period of PrEP usage.
+- *`PREP_JUST_STARTED`* - A boolean flag signifying whether an individual started using PrEP this time step.
+- *`LAST_PREP_USE_DATE`* - The date of an individual's most recent PrEP usage (at a time step granularity).
+- *`LAST_PREP_STOP_DATE`* - The stop date of an individual's most recent period of PrEP usage. Reset to None if they restart PrEP.
+- *`PREP_PAUSED`* - A boolean flag signifying whether an individual has paused their PrEP usage this time step due to ineligibility.
+- *`CONT_ON_PREP`* - A timedelta tracking the total length of continuous PrEP usage (at a time step granularity) of the current type of PrEP based on user intention. Breaks due to ineligibility do not count against continuity, but choosing to stop using PrEP does.
+- *`CONT_INTENT_ON_PREP`* - As `CONT_ON_PREP`, but this count will continue to increment even when an individual has paused PrEP usage due to ineligibility.
```

Comment:
```suggestion
- *`CONT_INTENT_ON_PREP`* - As `CONT_ON_PREP`, but this count will continue to increment even when an individual has paused PrEP usage due to ineligibility. This will reset if a person is diagnosed with HIV or is over 65, as above. 
```